eng/scripts/KillProcesses.ps1

```diff
@@ -38,6 +38,15 @@ function _killSeleniumTrackedProcesses() {
     }
 }
 
+function _listProcesses($processName) {
+    $processes = Get-WmiObject win32_process -Filter "name like '%$processName'" -ErrorAction SilentlyContinue;
+    if ($processes) {
+        Write-Host "These processes will be killed..."
+        $processes | select commandline
+    }
+}
+
+_listProcesses dotnet
```

Comment:
This could be done w/in the `_kill` function to cover everything we kill aka expect might hang or be left behind. Another option would be to list all executing processes to also list server processes we spin up when testing (say) Kestrel on full framework. Up to you&hellip;

Reply:
I'm explicitly going after the .NET processes. If someone has a use case for more info in the future they probably want different info. We know that logging the command line args of `dotnet` helps us so I'm going with that.